num_poses: 6
#pose0: [0, -0.785, 0.2, -2.1, 0, 1.571, 0.785]
#pose1: [-1.2, -0.785, 0, -2.1, 0, 1.571, 0.785]
#pose2: [1.2, -0.785, -0.4, -2.1, 0, 1.571, 0.785]
#pose3: [-1.2, -0.785, 0, -1.1, 0, 1.571, 0.785]
#pose4: [1.2, -0.285, -0.2, -2.1, 0, 2.571, 0.785]
#pose5: [-1.2, -0.785, 0, -2.1, 0, 1.571, 0.785]
#pose6: [1.2, -0.785, 0, -2.1, 0, 1.571, 0.785]

pose0: [-0.037, -0.886, -0.016, -2.398, -0.076, 1.58, 1.6]
pose1: [0.056, -0.641, -0.010, -2.472, 1.278, 1.980, 0.57]
pose2: [-1.20, -1.30, 0.41, -2.617, -0.321, 2.549, 0.726]
pose3: [-0.056, -1.390, -0.061, -1.977, -0.453, 3.216, -0.024]
pose4: [0.037, -0.028, -0.141, -2.229, -0.091, 1.161, 1.248]
pose5: [0, -0.785, 0.2, -2.1, 0, 1.571, 0.785]